tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4136,13 +4146,20 @@ public function test_is_oc_enabled( $pmc_enabled, $setting_enabled, $expected )
 			OC_Test_Helper::enable_oc();
 		}
 
-		$gateway = new WC_Stripe_UPE_Payment_Gateway();
+		// Reset the cached main gateway so that get_main_stripe_gateway() re-evaluates the
+		// current settings and picks UPE vs. OCS based on them.
+		$reflection = new ReflectionProperty( WC_Stripe::class, 'stripe_gateway' );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '4151', column '21'). (undefined)

(MissingImport)